### DISCUSSION TITLE
Welcome to INSAnonym-utils Discussions!

### DISCUSSION THREAD

**Discussion by danymat** [danymat/INSAnonym-utils#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions from GitHub as a place to interact with the participants of the INSAnonym competition.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

But first and foremost, have fun during the competition!

Daniel, aka danymat your fellow intern at LIFO.

**Comment by mxtla** [comment]
Bonjour, 
J'ai une question concernant le nombre maximum de soumission. Il est écrit que nous en avons 3 maximum, mais il y a une colonne "Publier" dans le tableau en dessous. Est-ce que nous pouvons en mettre autant que l'on veut mais en publier 3 maximum, ou est-ce que on peut en soumettre 3 et en publier une maximum ? 
Merci !

**Reply by danymat** [reply]
Bonjour et bienvenue sur le forum!
Concernant votre question:
- Vous pouvez soumettre au maximum 20 fichiers par équipe 
- Vous ne pouvez publier au maximum que 3 fichiers par équipe. Un fichier publié est un fichier qui pourra être attaqué et comptera dans le classement

J'espère que ça a répondu à votre question, n'hésitez pas à créer un post pour toute demande ultérieure ! 
En vous souhaitant bonne chance, 
Danymat.

**Reply by beng-git** [reply]
Il me semble que vous pouvez aussi décider de "dépublier" une soumission si jamais vous en avez une meilleure ensuite. Il n'y a donc pas de problème de publier vos soumissions au fur et à mesure si vous le souhaitez.
